### ISSUE TITLE
for multiclass label

### ISSUE THREAD

**Issue by Amg9794** [latishab/turnsense#4]
can this pretrained adopter be finetuned with 3 label ? or we need to fine-tune this adopter from scratch on 3 label data.

**Comment by latishab** [comment]
i think you can finetune with 3 label, you would need the data though.
